exercises/K-functions-parameters/exercise.js

```diff
@@ -1,6 +1,7 @@
 // Complete the function so that it takes input parameters
-function multiply() {
+function multiply(a,b) {
```

Comment:
As with the other statements, in a function declaration here it is also convention to place spaces between parameters:
```suggestion
function multiply(a, b) {
```